resources/windows/wix/service.component.wxs.erb

```diff
@@ -0,0 +1,58 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+  <%- get_services.each do |service| -%>
+    <%- service.options.each do |component| -%>
+    <DirectoryRef Id="<%= component[:directory_ref] %>">
+      <Component Id='<%= component[:id] %>' Guid="<%= component[:guid] %>" <%= "Win64=\"#{component[:win64]}\"" unless component[:win64].nil? %>>
+        <%- if component[:file][:id] == "NSSM" -%>
+        <CreateFolder />
```

Comment:
CreateFolder is only used if the folder has no contents in it, otherwise MSI would not create the folder on install, so I am not sure why it is here, when we know that the executable will be. In order to ensure we can remove it, we have to examine the directory specified in line 6 and see where that is located in our wxs files. If it is present in our wxs with files associated with it, then we can remove createfolder